app/validators/routing/new_routing_validator.py

```diff
@@ -1,76 +0,0 @@
-from app.validators.rules.rule_validator import RulesValidator
```

Comment:
Could this file have been renamed and the old one removed rather than updating the old with changes? As was done in other places in this PR

Reply:
As explained in Slack - file was renamed and old one remove, but Git diff doesn't understand it as such